_source/logzio_collections/_log-sources/salesforce-commerce-cloud.md

```diff
@@ -0,0 +1,140 @@
+-e LOGZIO_LISTENER_URL=<logzio_listener_url> \
+-e SFCC_SERVER_NAME=<your_sfcc_host> \
+-e SFCC_CLIENT_ID=<your_sfcc_client_id> \
+-e SFCC_LOG_SOURCE=operational
+-e SFCC_CLIENT_SECRET=<your_sfcc_client_secret> \
+logzio/sfcc-logs-fetcher:latest
+```
+
+| Parameter | Description | Required/Default |
+|---|---|---|
+|  `<<LOG-SHIPPING-TOKEN>>` | Your Logz.io account token. {% include log-shipping/log-shipping-token.html %}  | Required  |
+| `<<LOGZIO_LISTENER_URL>>` | {% include log-shipping/listener-var.md %} | Required |
+| SFCC_SERVER_NAME         | SFFC Server name from where you would like to collect logs.                                                                                                                                                                                                                                                                 |      Required |
+| SFCC_CLIENT_ID        | Client id related to the account that you need to send logs from. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                           |      Required |
+| SFCC_LOG_SOURCE    | Flag to represent which log types you would like to collect. `operational` for operational logs, `security` for security logs, `all` for both of them.                                                       |      Required |
+| SFCC_CLIENT_SECRET    | Client secret related to the account that you need to send logs from. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                       |      Required |
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
+
+If you still don't see your logs, see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
+
+</div>
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="config">
+
+The default configuration uses a Docker container with environment variables defined by a configuration file.
+
+<div class="tasklist">
+
+
+##### Create a local directory for this integration
+
+You will need a dedicated directory to use it as mounted directory for the Docker container of the Logz.io Salesforce Commerce Cloud.
+
+```shell
+mkdir sfcc-logs-fetcher
+cd sfcc-logs-fetcher
+```
+
+##### Create a configuration file
+
+In the directory created in the previous step, create a file `variables.env` as follows:
+
+```env
+# Environment file for Logz.io Logs for SalesForce Commerce Cloud
+LOGZIO_SHIPPING_TOKEN=<<LOG-SHIPPING-TOKEN>>
+SFCC_HOSTNAME=<<your_sfcc_host>>
```

Comment:
Hey! Great Job! Need to re name some parameter from SFCC_HOSTNAME to SFCC_SERVER_NAME

Reply:
Do we need to change the explanation as well? Instead of "<<your_sfcc_host>>"? (since it's not a host but a server)